doc/pachctl/pachctl_put-file.md

```diff
@@ -26,15 +26,20 @@ Put the contents of a directory as repo/commit/dir/file:
 	pachctl put-file -r repo commit -f dir
 
 Put the data from a URL as repo/commit/path:
-	pachctl put-file repo commit path -f http://host/url_path
+	pachctl put-file repo commit path -f http://host/path
 
-Put the data from a URL as repo/commit/url_path:
-	pachctl put-file repo commit -f http://host/url_path
+Put the data from a URL as repo/commit/path:
+	pachctl put-file repo commit -f http://host/path
 
 Put several files or URLs that are listed in file.
 Files and URLs should be newline delimited.
 	pachctl put-file repo commit -i file
 
+Put several files or URLs that are listed at URL.
+NOTE this URL can reference local files, so it could cause you to put sensitive
```

Comment:
I thought @jdoliner made it so that you can only do this with URLS for security reasons. Or did I misunderstand that PR?